SDK installation on React warnings

`npm i @dappykit/sdk`

```
test@madmax dappy-sdk-test % npm i npm i @dappykit/sdk
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: permissionless@0.1.49
npm warn Found: viem@2.21.52
npm warn node_modules/@dappykit/sdk/node_modules/viem
npm warn   viem@"^2.21.26" from @dappykit/sdk@3.0.0
npm warn   node_modules/@dappykit/sdk
npm warn     @dappykit/sdk@"*" from the root project
npm warn
npm warn Could not resolve dependency:
npm warn peer viem@">=2.14.1 <2.18.0" from permissionless@0.1.49
npm warn node_modules/@dappykit/sdk/node_modules/permissionless
npm warn   permissionless@"^0.1.28" from @dappykit/sdk@3.0.0
npm warn   node_modules/@dappykit/sdk
npm warn
npm warn Conflicting peer dependency: viem@2.17.11
npm warn node_modules/viem
npm warn   peer viem@">=2.14.1 <2.18.0" from permissionless@0.1.49
npm warn   node_modules/@dappykit/sdk/node_modules/permissionless
npm warn     permissionless@"^0.1.28" from @dappykit/sdk@3.0.0
npm warn     node_modules/@dappykit/sdk
⠋^C
test@madmax dappy-sdk-test % 
test@madmax dappy-sdk-test % 
test@madmax dappy-sdk-test % npm i @dappykit/sdk
npm warn ERESOLVE overriding peer dependency
npm warn While resolving: permissionless@0.1.49
npm warn Found: viem@2.21.52
npm warn node_modules/@dappykit/sdk/node_modules/viem
npm warn   viem@"^2.21.26" from @dappykit/sdk@3.0.0
npm warn   node_modules/@dappykit/sdk
npm warn     @dappykit/sdk@"*" from the root project
npm warn
npm warn Could not resolve dependency:
npm warn peer viem@">=2.14.1 <2.18.0" from permissionless@0.1.49
npm warn node_modules/@dappykit/sdk/node_modules/permissionless
npm warn   permissionless@"^0.1.28" from @dappykit/sdk@3.0.0
npm warn   node_modules/@dappykit/sdk
npm warn
npm warn Conflicting peer dependency: viem@2.17.11
npm warn node_modules/viem
npm warn   peer viem@">=2.14.1 <2.18.0" from permissionless@0.1.49
npm warn   node_modules/@dappykit/sdk/node_modules/permissionless
npm warn     permissionless@"^0.1.28" from @dappykit/sdk@3.0.0
npm warn     node_modules/@dappykit/sdk

added 16 packages, removed 226 packages, and audited 1360 packages in 4s

273 packages are looking for funding
  run `npm fund` for details

8 vulnerabilities (2 moderate, 6 high)

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

```

--- a/package.json
+++ b/package.json
@@ -72,12 +72,13 @@
     "ts-loader": "^9.5.1",
     "ts-node": "^10.9.2",
     "tslib": "^2.7.0",
-    "typescript": "^5.6.3",
+    "typescript": "^5.7.2",
     "webpack": "^5.95.0",
     "webpack-bundle-analyzer": "^4.10.2",
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
+    "abitype": "^1.0.7",
     "permissionless": "^0.2.21",
     "viem": "^2.21.22"
   },

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -31,6 +31,8 @@
     "rootDirs": [
       "src"
     ],
-    "outDir": "dist"
+    "outDir": "dist",
+    "sourceMap": true,
+    "declarationMap": false,
   }
 }

--- a/webpack.config.ts
+++ b/webpack.config.ts
@@ -54,7 +54,6 @@ const base = async (env?: Partial<WebpackEnvParams>): Promise<Configuration> => 
       )
     }
     // change node modules to browser modules according to packageJson.browser mapping
-    // eslint-disable-next-line guard-for-in
     const browserModuleMapping = PackageJson.browser as { [key: string]: string }
     // eslint-disable-next-line guard-for-in
     for (const nodeReference in browserModuleMapping) {
@@ -80,7 +79,8 @@ const base = async (env?: Partial<WebpackEnvParams>): Promise<Configuration> => 
     },
     bail: Boolean(isProduction),
     mode: env?.mode || 'development',
-    devtool: isProduction ? 'source-map' : 'cheap-module-source-map',
+    // disable for production until revised
+    devtool: isProduction ? false : 'cheap-module-source-map',
     entry,
     output: {
       path,

